### DISCUSSION TITLE
What happend with websockets

### DISCUSSION THREAD

**Discussion by paulfears** [MetaMask/snaps#1006] [Q&A] [answered]
I was using xrpl.js in snaps and it used to work, but now I get an error everytime xrpl.js tries to send a websocket request.

**Comment by FrederikBolding** [comment, marked as answer]
This is a known issue with the current MetaMask Flask release in the Chrome Store and will be fixed in the next release. See https://github.com/MetaMask/snaps-monorepo/issues/1004
